packages/tools/pie-css/test/scss/unit/functions/calculateUnitModifiers.spec.ts

```diff
@@ -0,0 +1,113 @@
+import {
+    describe,
+    it,
+    expect,
+} from 'vitest';
+
+import { compileCss, stripCSSWhitespace } from '../../../../utilities/compileCss';
+
+describe('functions.to-px', () => {
+    it('should render the expected CSS content', async () => {
```

Comment:
Do you need the async's here and throughout?

Reply:
I've no idea tbh - I'll try removing and running them locally again 👍

Reply:
Hey! I think if these are based off the existing tests it might have been left over from when I was inline reading and parsing CSS files and perhaps forgot to remove the now not necessary `async` works. 

If you remove these, please could you try removing from the other tests that don't need them? (there aren't many!)